daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -98,12 +98,11 @@ cmdTest numProcessors =
   where
     cmd = execTest
       <$> many inputFileOpt
+      <*> fmap (\x -> ColorTestResults x) colorOutput
       <*> junitOutput
       <*> optionsParser numProcessors optPackageName
-    junitOutput = optional $ strOption $
-        long "junit" <>
-        metavar "FILENAME" <>
-        help "Filename of JUnit output file"
+    junitOutput = optional $ strOption $ long "junit" <> metavar "FILENAME" <> help "Filename of JUnit output file"
```

Comment:
Why did you reformat this?

Reply:
Felt it was more readable in a line vs having it in multi line